packages/client/auth-core/server/index.ts

```diff
@@ -1 +0,0 @@
-export { verifyCrossmintSessionToken } from "../src/utils";
```

Comment:
I know we will focus on the client for this iterations, but I don't get why we remove this functions, unless there is already exported somehow in another place, or there's something I'm not seeing

Reply:
This was only re-exporting a util in the library for server code, but I was attempting to share the code for both instances and re-configured the exports so this would work in the same way without breaking anything. But now I noticed we'll need to keep them separate as `jsonwebtoken` doesn't work browser-side so I'll have to rethink this part